src/cli/ImportDirectoryTransactionalCommand.ts

```diff
@@ -0,0 +1,145 @@
+import * as path from "path";
+import * as memoize from "memoized-class-decorator";
+import fs = require("fs-extra");
+import byline = require("byline");
+import streamToPromise = require("stream-to-promise");
+
```

Comment:
not needed extra space

Reply:
I like to leave space between vendor libraries and internal files, but sure.